ivy/functional/frontends/numpy/ndarray/ndarray.py

```diff
@@ -57,3 +57,8 @@ def any(self, axis=None, out=None, keepdims=False, *, where=True):
 
     def argsort(self, *, axis=-1, kind=None, order=None):
         return np_frontend.argsort(self.data, axis, kind, order)
+
+    def prod(self , axis=None, dtype=None, out=None, keepdims=False, initial=1, where=True):
```

Comment:
(for lint fix) please format the parameters such that `param1, param2,...` (comma and spaces formatting)